rust-runtime/aws-smithy-http/src/byte_stream/bytestream_util.rs

```diff
@@ -24,19 +24,22 @@ use tokio_util::io::ReaderStream;
 pub struct PathBody {
     state: State,
     len: u64,
```

Comment:
can we rename this to `file_size` (now that `buffer_size` is also there it's slightly confusing)